Makefile

```diff
@@ -0,0 +1,70 @@
+# 
```

Comment:
I don't think we should wrap every single project in a Makefile. There are idiomatic patterns of how these tools should be used that we should follow, instead.